src/lib.rs

```diff
@@ -293,6 +293,14 @@ impl JayLink {
         let mut handle = usb_device
             .inner
             .open()
+            .map_err(|e| {
+                if cfg!(windows) && (e == rusb::Error::NotSupported || e == rusb::Error::NotFound) {
+                    info!(r"This error may be caused by not having the WinUSB driver installed.
+                    Use Zadig (https://zadig.akeo.ie/) to install the WinUSB driver for the J-Link device.
+                    This will replace the SEGGER J-Link driver!");
```

Comment:
I actually meant to put this in the error description, since users might not necessarily see `info!` (also, the error might be caught and handled by the caller, so we shouldn't use a side-effect for this message)

Reply:
Right, forgot that this was a library.